radio/src/opentx.cpp

```diff
@@ -357,6 +357,12 @@ void generalDefault()
   g_eeGeneral.pwrOnSpeed = 1; // 1 second
 #endif
 
+#if defined(RADIO_COMMANDO8)
+  g_eeGeneral.splashMode = 3;
```

Comment:
Please use `IFLIGHT_RELEASE` here. We should not have different defaults based on the radio. Using the proper tag allows you however to have the se defaults in your own builds & releases.